skyux-spa-ui/src/app/dogs/dogs-list.component.scss

```
@@ -12,7 +12,7 @@
 }
 
 .app-dog-card-title {
-  @extend .sky-page-heading;
+  @extend .sky-theme-font-heading-1;
```

Stylelint (17.12.0)

[error] Expected a placeholder selector (e.g. %placeholder) to be used in `@extend` (scss/at-extend-no-missing-placeholder)

(scss/at-extend-no-missing-placeholder)